recipe_scrapers/allrecipes.py

```diff
@@ -55,3 +67,115 @@ def cuisine(self):
 
     def category(self):
         return self.schema.category()
+
+
+class AllRecipesUser(AbstractScraper):
+    """Parse "unpublished" personal recipes on AllRecipes.com.
+
+    Unpublised recipes are not structured, unlike curated recipes.
+    Certain information is not always available, as it relies on what the
+    users provided.
+    """
+
+    @classmethod
+    def host(cls):
+        return "allrecipes.com/cook"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # meta contains servings and yields, as well as preparation, cooking,
+        # and total times
+        # Possible keys are 'servings', 'yield', 'cook', 'prep', 'total'
+        meta = zip(
+            self.soup.findAll("div", {"class": "recipe-meta-item-header"}),
+            self.soup.findAll("div", {"class": "recipe-meta-item-body"}),
+        )
+        self.meta = {
+            k.text.lower().strip(":"): normalize_string(v.text) for k, v in meta
```

Comment:
Nitpick: could use `rstrip`, mostly to hint to readers about the format of the headings. (`Foo:` and not `:foo:`)

Reply:
(pushed a commit to apply this suggested change, since doing that was roughly the same level-of-effort as adding a comment about it here.  feel free to review that in return)